pkgs/by-name/go/goat/package.nix

```diff
@@ -0,0 +1,26 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
```

Comment:
Switch the order of these
It goes lib, stdenv/your builder then fetchers, then everything else ASCII sorted

Reply:
Again, is this documented anywhere?

Reply:
The order is described in the last line of https://nixos.org/manual/nixpkgs/stable/#sec-syntax. The change seems to be correct somehow.